cdap-data-fabric-tests/src/test/java/co/cask/cdap/data/stream/StreamInputFormatTest.java

```diff
@@ -297,6 +299,7 @@ public void testStreamRecordReader() throws Exception {
     // one from 0 - some offset and one from offset - Long.MAX_VALUE.
     Configuration conf = new Configuration();
     TaskAttemptContext context = new TaskAttemptContextImpl(conf, new TaskAttemptID());
+    StreamInputFormat.setStreamId(conf, DUMMY_ID);
```

Comment:
users who? StreamInputFormat is not in api. Yes, it is because it tests StreamInputFormat in isolation.